catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -316,11 +316,13 @@ public void shutdown() {
     }
 
     public void checkForRollover() {
+        LOGGER.trace("checking for rollover");
```

Comment:
This happens once per second on a timer, it's not directly tied to incoming packets. It only tells us that the timer is running, which probably isn't that helpful. I'll drop it.